OBAKit/UI/Controls/StackedButton.swift

```diff
@@ -42,15 +42,28 @@ public class StackedButton: UIControl {
 
         return imageView
     }()
+    
+    override public var largeContentTitle: String? {
+        get { return self.textLabel.text }
+        set { /*empty*/ }
+    }
+    
+    override public var largeContentImage: UIImage? {
+        get { return self.imageView.image }
+        set { /*empty*/ }
+    }
 
     public override init(frame: CGRect) {
         super.init(frame: frame)
 
         isAccessibilityElement = true
         accessibilityTraits = .button
         isUserInteractionEnabled = true
-
+        
 		if #available(iOS 13, *) {
+            showsLargeContentViewer = true
```

Comment:
It looks like you have indentation with tabs instead of spaces turned on in Xcode.

<img width="912" alt="image" src="https://user-images.githubusercontent.com/2254/73140290-362ed900-402c-11ea-8cbd-48ea57f00da2.png">